envoy-control-services/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/services/transformers/InstanceMerger.kt

```diff
@@ -0,0 +1,35 @@
+package pl.allegro.tech.servicemesh.envoycontrol.services.transformers
+
+import pl.allegro.tech.servicemesh.envoycontrol.services.ServiceInstance
+import pl.allegro.tech.servicemesh.envoycontrol.services.ServiceInstances
+
+class InstanceMerger : ServiceInstancesTransformer {
```

Comment:
I see that we don't have tests for other transformers but could you add few test cases?

Reply:
est added